connector_magento/__manifest__.py

```diff
@@ -7,6 +7,7 @@
  'version': '10.0.1.0.0',
  'category': 'Connector',
  'depends': ['account',
+             'base_technical_user',
```

Comment:
Sorry it comes from https://github.com/OCA/server-tools/pull/858

It avoids to launch jobs with admin user and uses a dedicated technical company user.
It intends to replace https://github.com/OCA/connector-magento/pull/212/files#diff-131bcecd325b739cf356543a96bb2747R11 in a more generic way.